mlir/lib/Dialect/MIOpenOps/CppOutput/gridwise_convolution_implicit_gemm_v4r4.cpp

```diff
@@ -33,8 +33,6 @@ std::string resultStr;
 
 class TunableParameters : public TunableParametersBase {
 public:
-  TunableParameters() : TunableParametersBase("gridwise_convolution_implicit_gemm_v4r4.yaml") {}
-
   virtual void customInit() override {
     // parameters truly tunable.
     params["CK_PARAM_TUNABLE_GEMM_M_PER_BLOCK"] = 128;
```

Comment:
I have walked through those parameters with @asroy in a remote session and make sure those parameters can work for both forward and backward_data. But just to make sure, could you take a second look to sanity check this as well?